core/good_key.go

```diff
@@ -63,9 +64,80 @@ func GoodKey(key crypto.PublicKey) error {
 // GoodKeyECDSA determines if an ECDSA pubkey meets our requirements
 func GoodKeyECDSA(key ecdsa.PublicKey) (err error) {
 	log := blog.GetAuditLogger()
-	err = NotSupportedError("ECDSA keys not yet supported")
-	log.Debug(err.Error())
-	return
+
+	// Check the curve.
+	err = GoodCurve(key.Curve)
+	if err != nil {
+		return err
+	}
+
+	// Assuming a prime field since a) we are only allowing such curves
```

Comment:
Can you elaborate in this comment what impact it has on this particular line? In the unlikely event we start supporting non-prime curves in the future, what needs to change? We should probably document that next to the list of acceptable named curves, just to be safe.